ISISComputingGroup/ibex_gui
# Ticket 4165: Load script generator config from file

### Description of work

Loads information about the configuration of the script generator from a user defined python file.

### Ticket

See https://github.com/ISISComputingGroup/IBEX/issues/4165

### Acceptance criteria

To test:
* Comment out the hidden perspective
* Open the script generator perspective and confirm you can switch configs and the table changes
* Modify the parameters in the python code, restart the GUI and confirm the table updates appropriately.

### Unit tests
Little logic that could unit tested.

### System tests
Should probably do squish tests but ran out of time.

### Documentation
Ran out of time but documentation should be in https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/The-IBEX-Script-Generator

---

#### Code Review


### Final Steps
